README.md

```diff
@@ -27,3 +27,39 @@ SortAlgo
   └─visualizer/
   └─index
 ```
+
+## Features preview
+
+<img src="images/App.png" />
+
+### Choose the algorithm you want
+
+There are a list of algorithms available as well as its complexity evaluations and code available in `Java`, `JavaScript`, `Python` and `C/C++`
+
+Available algorithms are:
+* Bubble Sort
+* Insertion Sort
+* Selection Sort
+* Merge Sort
+* Quick Sort
+* Heap Sort
+* Shell Sort
+* Counting Sort
+* Radix Sort
+* Bucket Sort
```

Comment:
I think the features and what our app does is far more important than the project structure. Perhaps it can be below all these